### ISSUE TITLE
WordPress Pagination is not translated

### ISSUE THREAD

**Issue by S-mardii** [qtranslate/qtranslate-xt#1132]
I am trying to make sure that my whole WordPress page is localized by translating all the labels into the local language.

However, I encounter an issue in translating the "Next" and "Previous" labels on the pagination of the blog page.

PoEdit read the label of the translation, but the translated string is not applied.

Thank you in advance for the guidance.

```
function mytheme_posts_navigation() {
        the_posts_pagination( array(
                'mid_size'  => 3,
                'prev_text' => __( 'Previous', 'mytheme' ),
                'next_text' => __( 'Next', 'mytheme' )
            )
        );
    }
```

**Comment by herrvigg** [comment]
Yes, the basic functionality is independent from QTX, it's purely WordPress.  PO/MO files are for static labels for which the global WP language should apply. 

On top of that, QTX has a hook `gettext` that applies to all `__()` set of functions, so the translated string will be parsed by QTX...  You could have [ML tags](https://github.com/qtranslate/qtranslate-xt/wiki/Multilingual-fields) in the translated string, that would finally be translated by QTX. But this is not the nominal way to deal with translations and I don't like it. It was surely for retro-compatibility and other obscure reasons but his hook has been there forever so I don't dare removing it 😕
